malariagen_data/ag3.py

```diff
@@ -341,3 +350,78 @@ def _results_cache_add_analysis_params(self, params):
         super()._results_cache_add_analysis_params(params)
         # override parent class to add AIM analysis
         params["aim_analysis"] = self._aim_analysis
+
+    def load_inversion_tags(self, inversion: str):
```

Comment:
Can we give this on a docstring too, using the `@doc(...)` annotation? And `@check_types`.